cpp/oneapi/dal/table/backend/interop/host_homogen_table_adapter.hpp

```diff
@@ -108,12 +113,32 @@ class host_homogen_table_adapter : public daal::data_management::HomogenNumericT
                                      rw_mode_t rwflag,
                                      block_desc_t<BlockData>& block);
 
-    bool check_row_indexes_in_range(const block_info& info) const;
-    bool check_column_index_in_range(const block_info& info) const;
-
-    host_homogen_table_adapter(const homogen_table& table, status_t& stat);
+    bool check_row_indexes_in_range(const block_info& info) const {
```

Comment:
nip: what was the reason to return the definition of this function to the header file?

Reply:
Artefact of previous design idea, will move back